examples/tut_add-vectors.cpp

```diff
@@ -168,6 +168,18 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
 
   checkResult(c, N);
 //printResult(c, N);
+
+  std::cout << "\n Running RAJA CUDA explicit (2 blocks per SM) vector addition...\n";
+
+  // _rajacuda_explicit_vector_add_start
+  RAJA::forall<RAJA::cuda_exec_explicit<CUDA_BLOCK_SIZE/2, 2, false>>(RAJA::RangeSegment(0, N), 
```

Comment:
Please use a named bool variable for last template param for clarity; i.e.,
bool descriptive_name = false;
RAJA::forall<RAJA::cuda_exec_explicit<.....descriptive_name>>(...